<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>服务器控制台</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Close">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭服务器</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>服务器信息</string>
    </property>
    <widget class="QLabel" name="label_IPS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_PortS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>78</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>port：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SSS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>114</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>server status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_IP">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>40</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
text-decoration: underline;
font: 75 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>http://127.0.0.1/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Port">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>78</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 75 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SS">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>116</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Lucida Console&quot;;
color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Stop.</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>41</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <number>6000</number>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="value">
     <number>8888</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开启服务器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label__ln">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>173</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>已连接用户数</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>391</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_un">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>201</y>
      <width>48</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_uip">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>201</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ult">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>201</y>
      <width>64</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>上线时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ullt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>201</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>上次上线时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ulip">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>201</y>
      <width>79</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>上次上线IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_usernum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>173</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
